forge-gui/res/cardsfolder/upcoming/case_of_the_trampled_garden.txt

```diff
@@ -0,0 +1,13 @@
+Name:Case of the Trampled Garden
+ManaCost:2 G
+Types:Enchantment Case
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigPutCounter | TriggerDescription$ When this Case enters the battlefield, distribute two +1/+1 counters among one or two target creatures you control.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select one or two target creatures you control | CounterType$ P1P1 | CounterNum$ 2 | TargetMin$ 1 | TargetMax$ 2 | DividedAsYouChoose$ 2
+T:Mode$ Phase | Phase$ End of Turn | CheckSVar$ X | SVarCompare$ GE8 | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ SolveCase | TriggerDescription$ To solve — Creatures you control have total power 8 or greater. (If unsolved, solve at the beginning of your end step.)
```

Comment:
as TRT said in the other comment, this should not trigger if it's already been solved (but it will as submitted)